_sass/_basic.scss

```diff
@@ -115,3 +115,35 @@ th, td {
 tr:nth-child(even) {
     background-color: $teachla-tint;
 }
+
+h4{
+    font-size: 135%;
+}
+
+.team-content {
+    height: 100%;
+    max-width: 100%;
+    display: grid;
+    grid-template-columns: 33% 33% 33%;
+    //grid-template-columns: repeat(3, 1fr);
+
+    @media (max-width: 700px) {
+        height: 100%;
+        max-width: 100%;
+        display: grid;
+        grid-template-columns: 100%;
+    }
+}
+
+.member-card {
+    display: flex;
+    justify-content: flex-start;
+    width: 100%;
+    height: 100%;
+}
+
+.member-info {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+}
```

Comment:
If you rename these classes slightly, they can be more reusable (and would make more sense) - imagine class names as functional, or describing what they do (i.e. `.row` and `.column`) rather than purely describing what they're used for.